stack/app/variables.tf

```diff
@@ -297,6 +297,16 @@ variable "rds_subnet_group_name" {
   default = null
 }
 
+variable "rds_backup_retention_period" {
+  default = 7
+  type    = number
+}
+
+variable "rds_max_allocated_storage" {
+  type    = number
+  default = 0
```

Comment:
Suggestion: Should we default to auto-scaling up to 20 GiB? +10 GiB won't hurt, and we have a bit of a safe-guard.